### PR TITLE
[shell] fs log purge

### PR DESCRIPTION
**Pull request by kmlebedev**
# What problem are we solving?

one command for delete old logs

# How are we solving the problem?

add shell command

# How is the PR tested?

localy
```
> fs.log.purge -v -dayAgo 1
skip 2024-02-28
> fs.log.purge -v -dayAgo 0
delete 2024-02-28
> fs.ls -a /topics/.system/log
> fs.ls -a /topics/.system

```

# Checks
- [ ] I have added unit tests if possible.
- [ ] I will add related wiki document changes and link to this PR after merging.

### CODE REVIEW

File: weed/shell/command_fs_log.go [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+func (c *commandFsLogPurge) Help() string {
+	return `purge filer logs
+
+	fs.log.purge [-v] [-modifyDayAgo 365]
+`
+}
+
+func (c *commandFsLogPurge) Do(args []string, commandEnv *CommandEnv, writer io.Writer) (err error) {
+	fsLogPurgeCommand := flag.NewFlagSet(c.Name(), flag.ContinueOnError)
+	modifyDayAgo := fsLogPurgeCommand.Uint("modifyDayAgo", 365, "purge logs older than N days")
```
Comment (chrislusf) [inline weed/shell/command_fs_log.go]: use `timeAgo` as other places?

logs can not be modified anyway.

Comment (kmlebedev) [reply weed/shell/command_fs_log.go]: use dayAgo

**Comment by kmlebedev** [comment]
@chrislusf  done

File: weed/shell/command_fs_log.go [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+func (c *commandFsLogPurge) Help() string {
+	return `purge filer logs
+
+	fs.log.purge [-v] [-modifyDayAgo 365]
+`
+}
+
+func (c *commandFsLogPurge) Do(args []string, commandEnv *CommandEnv, writer io.Writer) (err error) {
+	fsLogPurgeCommand := flag.NewFlagSet(c.Name(), flag.ContinueOnError)
+	dayAgo := fsLogPurgeCommand.Uint("dayAgo", 365, "purge logs older N day")
```
Comment (chrislusf) [inline weed/shell/command_fs_log.go]: Nit: `dayAgo` => `daysAgo`.

Comment (chrislusf) [reply weed/shell/command_fs_log.go]: "purge logs older N day" => "purge logs older than N days"

Comment (kmlebedev) [reply weed/shell/command_fs_log.go]: @chrislusf done

**Pull request merged**